README.md

```diff
@@ -95,3 +95,24 @@ and/or with `docker-compose`:
 ```
 docker-compose build && docker-compose run --rm app go test ./...
 ```
+
+### Release a new version
+
+1. Update go dependencies (`bitrise run godeps-update`)
+1. PR & merge these changes to the `master` branch
```

Comment:
nop, markdown allows you to list with `1. 1. 1.` and will generate a proper list like this

1. first
1. second
1. third